src/reactviews/pages/ExecutionPlan/executionPlanGraph.tsx

```diff
@@ -0,0 +1,203 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { useContext, useEffect, useState } from 'react';
+import { ExecutionPlanContext } from "./executionPlanStateProvider";
+import * as utils from './queryPlanSetup';
+import * as azdataGraph from 'azdataGraph/dist/build';
+import 'azdataGraph/src/css/common.css';

```

Comment:
this seems odd pattern to me.  Shouldn't the control import its own CSS files.

Reply:
same reason as the above comment, this was the only way to load in stylings.

Reply:
hmm, is this something that should be fixed in the control?  It's fine to leave for initial check-in but would be good to fix at some point.